libs/auth-api-lib/src/lib/user-notification/apiConfiguration.ts

```diff
@@ -0,0 +1,26 @@
+import { Provider } from '@nestjs/common'
+
+import { createEnhancedFetch } from '@island.is/clients/middlewares'
+import { ConfigType, LazyDuringDevScope } from '@island.is/nest/config'
+import { Configuration, NotificationsApi } from './generated/fetch'
+
+import { DelegationApiUserSystemNotificationConfig } from './user-system-notification.config'
+
+export const NotificationsApiProvider: Provider<NotificationsApi> = {
+  provide: NotificationsApi,
+  scope: LazyDuringDevScope,
+  useFactory: (
+    config: ConfigType<typeof DelegationApiUserSystemNotificationConfig>,
+  ) =>
+    new NotificationsApi(
+      new Configuration({
+        fetchApi: createEnhancedFetch({
+          name: 'delegation-api-system-user-notification',
+          organizationSlug: 'stafraent-island',
+          autoAuth: undefined,
+        }),
+        basePath: `${config.basePath}/v1`,
```

Comment:
We shouldn't need to configure the `v1` segment of the url. I looked at the open api generation of the `user-notifications` service and I see that the openapi isn't generating the paths with versions due to `enableVersioning` missing the `buildOpenApi`. So you should just enable that and then the open api document will contain versioned paths and the generated client code will contain the version.